openquake/commonlib/oqvalidation.py

```diff
@@ -789,8 +789,7 @@ def is_valid_export_dir(self):
         and the user must have the permission to write on it.
         """
         if self.export_dir and not os.path.isabs(self.export_dir):
-            self.export_dir = os.path.normpath(
-                os.path.join(self.input_dir, self.export_dir))
+            self.export_dir = os.getcwd()
```

Comment:
I would add a warning is the `export_dir` is not set (like it was in the past):


```suggestion
            self.export_dir = os.getcwd()
            logging.warn('export_dir not specified. Using export_dir=%s'
                         % self.export_dir)
```

Reply:
I am adding it